CMakeLists.txt

```
@@ -204,8 +205,18 @@ if(NOT SPFFT_MKL AND ${SPFFT_FFTW_LIB} STREQUAL "AUTO" OR ${SPFFT_FFTW_LIB} STRE
 	endif()
 endif()
 
+# Look for NVPL
+if(NOT SPFFT_MKL AND ${SPFFT_FFTW_LIB} STREQUAL "AUTO" OR ${SPFFT_FFTW_LIB} STREQUAL "NVPL")
```

```suggestion
if(NOT SPFFT_MKL AND NOT SPFFT_ARMPL AND ${SPFFT_FFTW_LIB} STREQUAL "AUTO" OR ${SPFFT_FFTW_LIB} STREQUAL "NVPL")
```
In case both ARMPL and NVPL are found, we should not link to both.

---

Good catch! Thanks